### DISCUSSION TITLE
An internal error has occurred - attempt to perform arithmetic on field

### DISCUSSION THREAD

**Discussion by phippster** [rsjaffe/MIDI2LR#677] [Problems]
Since I upgraded to the latest version 5.4.0.0 I keep getting an infinite number of warning messages that fill my entire screen.  I'm really bummed as I had to disconnect MIDI2LR in order to edit my photos at this last professional gig for a festival.  The errors happen on a variety of knobs and dials that I have configured for things like Exposure adjustment, shadows, etc. 

Error reads "An internal error has occurred. - ?:0 attempt to perform arithmetic on field '?' (a nil value)"

Please please please can we get this addressed.  Been using MIDI2LR for years and this is really the first show stopping problem which has destroyed my workflow.  Amazing tool but man this bug is bad.

<img width="433" alt="Screen Shot 2022-07-30 at 12 57 53 PM" src="https://user-images.githubusercontent.com/1810557/182964081-7c4b3ee8-d608-4da1-b6be-52ecf82cec54.png">

**Comment by rsjaffe** [comment]
Do you have a profile assigned to masking?

**Reply by phippster** [reply]
Not that I'm aware of. Here's my mappings XML file
[Mappings 8.5.22.xml.txt](https://github.com/rsjaffe/MIDI2LR/files/9276285/Mappings.8.5.22.xml.txt)
:

**Reply by phippster** [reply]
Just verified there are no Masking profiles.

**Reply by phippster** [reply]
Any progress at finding and repairing this bug?  MIDI2LR is unusable right now.
